examples/PlotSpeedTest.py

```diff
@@ -5,53 +5,54 @@
 """
 
 ## Add path to library (just for examples; you do not need this)
```

Comment:
This comment is referring to `import initExample` and should be kept above that line.